rest_framework/views.py

```diff
@@ -118,8 +119,19 @@ def as_view(cls, **initkwargs):
         This allows us to discover information about the view when we do URL
         reverse lookups.  Used for breadcrumb generation.
         """
+        if isinstance(getattr(cls, 'queryset', None), models.query.QuerySet):
+            def force_evaluation():
+                raise AssertionError(
```

Comment:
@berkerpeksag Whoops - missed that when merging, but now addressed. Thanks for the input! :)